types/schema/data/Application.ts

```diff
@@ -49,27 +49,69 @@ export interface ApplicationFee {
    * @description Breakdown of calculated fee in GBP by category of development, based on the scales defined in The Town and Country Planning Regulations https://www.legislation.gov.uk/uksi/2012/2920/schedule/1/part/2
    */
   category?: {
+    /**
+     * @description Category 1 - New homes
+     */
     one?: number;
-    oneToFour?: number;
```

Comment:
Going through the service again, I realised that these ranges (any variable with `To`) are actually booleans used for internal automations only, so they don't need to be included here !